src/tests/exploration_rankscore/test_assemble_unique_variant_id.py

```diff
@@ -0,0 +1,38 @@
+import pandas as pd
+import pytest as pt
```

Comment:
```suggestion
import pandas
import pytest
```

I think this improves readability and reduces cognitive load